cmd/notice-file-generator/dependency.go

```diff
@@ -346,12 +347,41 @@ func PopulateGoDependencies(config *Config) ([]Dependency, error) {
 	return dependencies, nil
 }
 
+func PopulatePythonDependencies(config *Config) ([]Dependency, error) {
+	dependencies := []Dependency{}
+	inFile, err := os.Open(config.Path + "/Pipfile")
+	if err != nil {
+		log.Fatalf("Invalid pipfile. %v", err)
+	}
+	defer inFile.Close()
+	scanner := bufio.NewScanner(inFile)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "# Repo:") {
```

Comment:
I think instead of nested ifs which make it harder to read, it will better to do the opposite. `if !strings.HasPrefix(..), continue` etc. etc. This will make this block of code a bit more readable

Reply:
I was lazy while doing that, and will do it with regex checks to have only one if condition.

Reply:
Code simplified to check and parse variables at the same time by using regex.